### ISSUE TITLE
I, as a user, want to know the goals of a game so I can contribute towards those goals, understand where the game's producers are, and the next steps such as what type of character or quests will be created.

### ISSUE THREAD

**Issue by getuliosa** [niksonleonard/the-delightful-letters-mess-game#24]

**Comment by getuliosa** [comment]
@niksonleonard @RaniB29 

META 1

A primeira meta de contribuição do jogo Mamu da Pombis será de R$ 3.000,00.

-Criação da história
-Criação do primeiro ambiente e primeiros personagens.
-Fornecer Informações i sobre saúde para os jogadores.
– Publicação o jogo nas lojas da App Store, Google Play e Meta Store.
– Infraestrutura de servidores e hospedagem necessária para manter o jogo em execução.


META 2

A segunda meta de contribuição será de R$ 8.000,00, e com ela vamos ter:

– Criação de novos ambientes-
-Criação de  novas missões
–Criação de novos roteiros e histórias
-Criação de novos personagens

**Comment by niksonleonard** [comment]
@getuliosa it is not clear to me if these Goals Topics are suggestions or it is just a template to be filled, could you make it more clear how who is reading it should interpret? 

One thing about it is there is information that is not valid as Value Proposition for a Player, maybe just as Donator. So I will make a suggestion but it will be strongly connected to the discussion that we will have next week. For now I will leave a suggestion draft:

Goal 1 - R$ 3.700,00

- One Playable character
- Only one scenario to play without Story Campaign 
- Global Score record when a Player
- Available to download on Google Play and Meta Store
- A Donate way directly from the game
...

Goal 2 - R$ 10.000,00

- Campaign Mode
- Power ups to Playable character to give tons of strategies 
- Skin store for the Player
- Mission with Stamps as reward based in Score
- Album of Collectible Stamp
- Available to download on Apple App Store

...

Goal 3 - R$ 45.000,00

- Customizable Player
- Multiplayer Mode with exclusive mission and Stamps for Album Collection
- Social exchange for gist
- Regional Rank with Season reseting
...

Goal 4 - 700.000,00 

- Institution of Pombas and Klumfy as Game School Cooperative
...


As I mentioned before it is just a suggestion but I put some ideas that could be interesting think about as milestones and goals.

**Comment by getuliosa** [comment]
That's right, it's an extension of the one I had put on Google Docs, and it was very detailed and in text form, so you asked me to do it on request.

I look for some references in game financing, and I made our own, but as I said before, in these first ones I would really need your help, so yes, it was something suggestive to give a direction, so that the goal could be aligned with the moment real sense of what's being created between you and Alan, and I didn't leave something out, or include something that didn't need it.

We will follow yours as a reference.
I thank.
